database.rules.json

```diff
@@ -2,10 +2,22 @@
   "rules": {
     ".read": false,
     ".write": false,
-    "messages": {
+    "games": {
       "123": {
```

Comment:
This is still just for testing purposes, right?

Reply:
Yes, in the future it should be set to `$game_id`, but for now there can only be one game session `123`